tests/backward-compat/upgrade/src/test/groovy/org/apache/bookkeeper/tests/backwardcompat/TestCompatUpgrade.groovy

```diff
@@ -202,7 +202,70 @@ class TestCompatUpgrade {
     }
 
     @Test
-    public void test472toCurrentMaster() throws Exception {
-        testUpgrade("4.7.2", System.getProperty("currentVersion"))
+    public void test472to480() throws Exception {
+        testUpgrade("4.7.2", "4.8.0")
+    }
+
+    @Test
+    public void test480to481() throws Exception {
+        testUpgrade("4.8.0", "4.8.1")
```

Comment:
can we drop this old stuff?

Reply:
@eolivelli I added issue https://github.com/apache/bookkeeper/issues/2721 to track this request. I think it makes sense to do that i separate PR/have a larger group of committers to chirp in on how far back eh backwards compat should go